pages/common/dotenvx.md

```diff
@@ -0,0 +1,32 @@
+# dotenvx
+
+> A better `dotenv`, from the creator of `dotenv`.
+> More information: <https://dotenvx.com/docs>.
+
+- Run a command with environment variables from a .env file:
+
+`dotenvx run -- {{command}}`
+
+- Run a command with environment variables from a specific .env file:
+
+`dotenvx run -f {{path/to/.env}} -- {{command}}`
+
+- Set an environment variable with encryption:
+
+`dotenvx set {{KEY}} {{value}}`
+
+- Set an environment variable without encryption:
+
+`dotenvx set {{KEY}} {{value}} --plain`
+
+- Return environment variables defined in a .env file:
+
+`dotenvx get`
+
+- Return the value of an environment variable defined in a .env file:
+
+`dotenvx get {{KEY}}`
+
+- Return all environment variables from .env files and OS:
```

Comment:
```suggestion
- Run a command with environment variables from a `.env` file:

`dotenvx run -- {{command}}`

- Run a command with environment variables from a specific `.env` file:

`dotenvx run -f {{path/to/.env}} -- {{command}}`

- Set an environment variable with encryption:

`dotenvx set {{KEY}} {{value}}`

- Set an environment variable without encryption:

`dotenvx set {{KEY}} {{value}} --plain`

- Return environment variables defined in a `.env` file:

`dotenvx get`

- Return the value of an environment variable defined in a `.env` file:

`dotenvx get {{KEY}}`

- Return all environment variables from `.env` files and OS:
```
https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#special-cases

Reply:
That's great. I was still struggling with some style problems before, so I referred to some other pages, but it seemed incomplete. You provided a good reference. Thank you very much. 😊